build/install

```diff
@@ -32,6 +32,11 @@ if [ "${PHP_FULL_VERSION:0:3}" == "5.3" ]; then
 	exit 1
 fi
 
+if [ "${PHP_FULL_VERSION:0:3}" == "5.4" ]; then
+	echo "php 5.4 is no longer supported"
```

Comment:
Is there any reason for not supporting 5.4 ? Like something that doesn't work with php 5.4 ?